### PR TITLE
feat: 991 update headshots on team page

### PR DESCRIPTION
**Pull request by haveanicedavid**
## Description

Closes: #991 

I noticed doing this story that the images were also not optimized. We weren't using the `TeamSection` anywhere but `www`, so I moved it along with respective components to live in /www, and converted the images to `SanityImage` (along with the type updates)

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood` and `hambach` (see below)

#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

2. If your branch does have merge commits:

    a. Pull latest `master`, `hambach` and `redwood` branches

    b. Create new branches for backports and merge `master` (replace `<PR#>` with your PR #)
    ```
    git checkout -b hambach-backport-<PR#> hambach
    git merge master
    git push origin hambach-backport-<PR#>
    git checkout -b redwood-backport-<PR#> redwood
    git merge master
    git push origin redwood-backport-<PR#>`
    ```

    c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.
  

### How to test

1. go to the `/team` page on the website preview
2. confirm it loads and looks correct

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-components/src/components/header/HeaderMenuHover.tsx [hunk 1]
```diff
@@ -97,11 +95,12 @@ const HeaderMenuHover: React.FC<HeaderMenuHoverProps> = ({
...
-        dropdownColor={
-          color === theme.palette.primary.light
-            ? theme.palette.secondary.main
-            : theme.palette.secondary.contrastText
-        }
+        // dropdownColor={
+        //   color === theme.palette.primary.light
+        //     ? theme.palette.secondary.main
+        //     : theme.palette.secondary.contrastText
+        // }
```
Comment (mhagel) [inline web-components/src/components/header/HeaderMenuHover.tsx:102]: ✂️ ?

File: web-www/src/graphql/teamMemberFields.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { graphql } from 'gatsby';
+
+export const teamSectionFields = graphql`
```
Comment (mhagel) [inline web-www/src/graphql/teamMemberFields.ts:3]:
```suggestion
export const teamMemberFields = graphql`
```
?

**Review by mhagel — APPROVED** [review]
Just a few nits. Nice work!

File: web-www/src/graphql/teamMemberFields.ts (hunk 2 above)
Comment (haveanicedavid) [reply web-www/src/graphql/teamMemberFields.ts:3]: oh good catch

**Review by wgwz — APPROVED** [review]
LGTM! while i don't understand precisely how this works, i feel that i would stand a chance at figuring it out with the way you've structured the code, i.e. by using `TeamItem` and `TeamSection` modules. I'm curious to understand more about how gatsby and graphql work but that's a convo for another time. I really like how the images look as well, nice work!

**Comment by haveanicedavid** [comment]
@Mergifyio backport redwood hambach v4

**Pull request merged**